docs/sources/flow/_index.md

```diff
@@ -90,6 +90,6 @@ This feature is experimental, and it doesn't support all River components.
 [Concepts]: {{< relref "./concepts/" >}}
 [Getting started]: {{< relref "./getting-started/" >}}
 [Tutorials]: {{< relref "./tutorials/ ">}}
-[Configuration language]: {{< relref "./config-language/" >}}
+[Configuration language]: {{< relref "./concepts/config-language/" >}}
```

Comment:
Since Configuration language is part of concepts now, we can remove it from "next steps" which already tells you to read the concepts.